### PR TITLE
[IIIF-267] [IIIF-274] [IIIF 276] Fix docker bucketeer build

### PR DESCRIPTION
**Pull request by ksclarke**
One issue left to fix before this is merged.

### CODE REVIEW

**Comment by ksclarke** [comment]
Okay, this is ready now -- it's building successfully on my machine and on the CodeBuild machine now.

File: README.md [hunk 1]
```diff
@@ -25,7 +34,10 @@ This will run the container in the background until `docker stop bucketeer` is c
 You can also run the container with environmental variables supplied on the command line like:
 
     docker run -d -p 8181:8181 \
-      -e "HTTP_PORT=8181" \
+      -e "BUCKETEER_S3_BUCKET=YOUR_VALUE_HERE" \
```
Comment (hardyoyo) [inline README.md]: will these work, without ENV added in the Dockerfile for each ARG?

Comment (ksclarke) [reply README.md]: Yes, at runtime we use a config file mechanism that will take ENV variables and write them into the config file that's put on the container.

Comment (ksclarke) [reply README.md]: Specifically: https://github.com/UCLALibrary/docker-jp2-bucketeer/blob/master/docker-entrypoint.sh#L23

Comment (cachemeoutside) [reply README.md]: Just a note, whenever storing a default ENV in the Dockerfile, this stores it in the image's metadata.

e.g.

`
docker inspect uclalibrary/cantaloupe-ucla
...
             "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "CANTALOUPE_VERSION=dev",
                "CONFIG_FILE=/etc/cantaloupe.properties",
                "LD_LIBRARY_PATH=/usr/lib/jni:"
            ],
...
`

Comment (cachemeoutside) [reply README.md]: So if we have any integration tests that requires access to S3, that keys should *not* be in the Dockerfile itself, but set at Travis's ENV.

Comment (ksclarke) [reply README.md]: Yes, which is why we're doing this at runtime (so they don't go into the container). No ENVs with these values are set in the Dockerfile.

**Pull request merged**